Fixes handling bitwise operators for z3 model checker (#13939)
ethereum/solidity: Changelog.md

 * SMTChecker: Fix internal error when deleting struct member of function type.
 * SMTChecker: Fix internal error when using user-defined types as mapping indices or struct members.
 * SMTChecker: Improved readability for large integers that are powers of two or almost powers of two in error messages.

>  * SMTChecker: Fix internal error caused by unhandled ``z3`` expressions that come from the solver when bitwise operators are used.

Reviewer: @pgebal this should be moved to the top of the file under 0.8.19

>  * SMTChecker: Fix compiler error caused by unhandled ``z3`` expressions that come up when bitwise operators are used in solidity code.

Reviewer: Nit:
```suggestion
 * SMTChecker: Fix internal error caused by unhandled ``z3`` expressions that come from the solver when bitwise operators are used.
```

>  * SMTChecker: Fix compiler error caused by unhandled ``z3`` expressions that come up when bitwise operators are used in solidity code.

Reviewer 2: This is changelog for 0.8.18, which has already been released. Please move to 0.8.19 section.
